index.html

```diff
@@ -31,7 +30,126 @@ <h1 class="black-text-header">Hey there. I’m<br> Jorge<br><span>I’m a softwa
+        <div class="multi-post post-two">
+          <h3 class="multi-post-header">Multi-Post Stories<br>Gain+Glory</h3>
+          <ul class="multi-post-skills">
+            <li class="multi-post-skill">Ruby on rails</li>
+            <li class="multi-post-skill">css</li>
+            <li class="multi-post-skill">JavScript</li>
+            <li class="multi-post-skill">html</li>
+          </ul>
+          <button class="work-button">See Project</button>
+        </div>
+      </div>
+      <div class="recent-work three">
+        <div class="multi-post post-three">
+          <h3 class="multi-post-header">Multi-Post Stories<br>Gain+Glory</h3>
+          <ul class="multi-post-skills">
+            <li class="multi-post-skill">Ruby on rails</li>
+            <li class="multi-post-skill">css</li>
+            <li class="multi-post-skill">JavScript</li>
+            <li class="multi-post-skill">html</li>
+          </ul>
+          <button class="work-button">See Project</button>
+        </div>
+      </div>
+      <div class="recent-work four">
+        <div class="multi-post post-four">
+          <h3 class="multi-post-header">Multi-Post Stories<br>Gain+Glory</h3>
+          <ul class="multi-post-skills">
+            <li class="multi-post-skill">Ruby on rails</li>
+            <li class="multi-post-skill">css</li>
+            <li class="multi-post-skill">JavScript</li>
+            <li class="multi-post-skill">html</li>
+          </ul>
+          <button class="work-button">See Project</button>
+        </div>
+      </div>
+      <div class="recent-work five">
+        <div class="multi-post post-five">
+          <h3 class="multi-post-header">Multi-Post Stories<br>Gain+Glory</h3>
+          <ul class="multi-post-skills">
+            <li class="multi-post-skill">Ruby on rails</li>
+            <li class="multi-post-skill">css</li>
+            <li class="multi-post-skill">JavScript</li>
+            <li class="multi-post-skill">html</li>
+          </ul>
+          <button class="work-button">See Project</button>
+        </div>
+      </div>
+      <div class="recent-work six">
+        <div class="multi-post post-six">
+          <h3 class="multi-post-header">Multi-Post Stories<br>Gain+Glory</h3>
+          <ul class="multi-post-skills">
+            <li class="multi-post-skill">Ruby on rails</li>
+            <li class="multi-post-skill">css</li>
+            <li class="multi-post-skill">JavScript</li>
+            <li class="multi-post-skill">html</li>
+          </ul>
+          <button class="work-button">See Project</button>
+        </div>
+      </div>
+    </section>
+    <h2 class="about-me-header">About me</h2>
+    <p class="about-me-paragraph-mobile"> I can help you build a product,<br>feature or website Look through<br>some of
+      my work and
+      experience! If<br>you like what you see and have a<br>project you need coded,<br>don’t hesitate to contact me.</p>
+    <button class="about-me-button">Get My Resume</button>
```

Comment:
- [ ] Good job so far by implementing the about section but I think this part is a part of the about section, please kindly add it to the about `section` tag 😁😁✌️